src/pages/hackaviz.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import useHackaviz from "../hooks/useHackaviz";
+
+const Hackaviz = () => {
+  const currentYear = new Date().getFullYear();
+  const previousYear = currentYear - 1;
+  const isCurrent = (year) => year === currentYear.toString();
+  const hackavizs = useHackaviz();
+
+  let redirectUrl = "/";
```

Comment:
Le composant Hackaviz ne devrait être utilisé que pour faire de la présentation en fonction de ses props et non pas avir de la logique de redirection

Reply:
Pour indiquer la page où l'on est redirigé après avoir cliqué sur hackaviz dans le menu, utiliser home_content.csv https://github.com/ToulouseDataViz/website/blob/main/src/data/home_content.csv et changer le slug associé au hackaviz vers la page de son choix -> par exemple /hackaviz/hackaviz2021-contest

Reply:
Ou alors à la limite ajouter cette logique ici https://github.com/ToulouseDataViz/website/blob/main/src/pages/index.js#L31
Mais je pense que le plus simple c'est de l'avoir dans home_content, parce que ce sera changé max 2 fois par an, c'est plus simple.

Reply:
Nb, dans d'autres cas, si besoin de naviguer vers une autre page en fonction d'une action, il y a le helper de gastby `navigate` : https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/#how-to-use-the-navigate-helper-function